<!DOCTYPE HTML>
<!--
	Spectral by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Calciomics</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="static/calciomics/assets/css/main.css" />
		<noscript><link rel="stylesheet" href="static/calciomics/assets/css/noscript.css" /></noscript>
	</head>
	<body class="landing is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header" class="alt">
						<h1><a href="https://calciomics.org">Calciomics</a></h1>
						<nav id="nav">
							<ul>
								<li class="special">
									<a href="#menu" class="menuToggle"><span>Menu</span></a>
									<div id="menu">
										<ul>
											<li><a href="https://calciomics.org">Home</a></li>
											<li><a href="https://calciomics.org/resources">Resources</a></li>
											<li><a href="https://calciomics.org/register">Sign Up</a></li>
											<li><a href="https://calciomics.org/login">Log In</a></li>
										</ul>
									</div>
								</li>
							</ul>
						</nav>
					</header>

				<!-- Banner -->
					<section id="banner">
						<div class="inner">
							<h2>Calciomics</h2>
						</div>
						<a href="#one" class="more scrolly">Learn More</a>
					</section>

				<!-- Main -->
				<article id="main">
					<section class="wrapper style5">
						<div class="inner">

							<p>The objective of this research is to develop
								bioinformatics methods to predict and identify different classes of calcium
								binding sites in proteins based on key factors that contribute to
								calcium-binding affinity and the selectivity of proteins and calcium-dependent
								conformational change.</p>

							<p>In collaboration with faculty in the Georgia State University
								Department of Mathematics and Statistics, algorithms for predicting calcium
								binding sites based on structural and genomic information have been established
								using geometric, graphing, and statistical algorithms along with a web server
								containing up-to-date sequence, structure and literature information about
								calcium and its binding proteins in chemical, biological, and
								biomineralization. The Calcium Bank also provides putative calcium binding
								sites predicted in different genomes.</p>

							<hr />

							<h4>Calcium Pattern Search</h4>
							<p><a href="https://shenghuixue.github.io/CAPS/index.html">CaPS(Calcium Pattern Search</a> provides a list of Ca(II)-binding A sequence patterns that indicate the presence of a Ca(II)-binding motif. To date, we have developed sequence patterns, or signatures, for canonical EF-hand, pseudo EF-hand, and baterial EF-hand. We expect to significantly expand this list in the future to indlude non-contiguous (binding motif includes A residues distant from each other in the primary structure) patterns. To check a sequence for the presence of a Ca(II)-binding signature click on the CaPS link and enter your sequence for analysis.</p>

							<h4>MUG<sup>π</sup></h4>
							<p><a href="https://calciomics.org/login">MUG<sup>π</sup></a> is a web-based algorithm for processing x-ray and NMR protein data using bio-python. MUGπ creates a protein topology network using bio-python and NetworkX. A NetworkX topology of oxygen and carbon maximum cliques is used to filter protein features for calcium-binding site identification.</p>

							<p>This is the latest iteration of software developed in our labs to identify Ca2+-binding sites in protein structure files. The first iteration of the program, GG, was a graph-theoretic and geometric analysis program developed by Hai Deng, Guan Tao Chen, Wei Yang, and Jenny J Yang. The program was used for predicting Ca(II)-binding sites in protein based on the geometric information. GG was later extended in the MUG (Multiple Geometries) program, a prediction algorithm capable of identifying calcium-binding sites in proteins with atomic resolution. MUG worked by first identifying all possible oxygen clusters from maximal cliques, and then calculating a calcium center (CC) for each cluster corresponding to the potential Ca2+ position, located to maximally regularize the structure of the (cluster, CC) pair. The structure was then inspected by geometric filters. Any unqualified (cluster, CC) pairs were further handled by recursively removing oxygen atoms and relocating the CC until its structure was either qualified or contained fewer than four ligand atoms. Ligand coordination was then determined for qualified structures. Immediately following the release of MUG, MUGC was developed to recognize calcium-binding sites without explicit reference to side-chain oxygen ligand coordinates. By using second shell carbon atoms, main-chain oxygen atoms, center of mass of side-chain, and graph theory, it was able to predict calcium-binding sites exhibiting insignificant conformational change with high sensitivity and selectivity based on X-ray structures or NMR structures.</p>

							<p>The new, advanced MUGπ extends the original algorithms efforts to identify oxygen clusters, combined with the identification of carbon clusters, into a single application. MUGπ was developed by M.G. Mashiku under the supervision of Dr. Suranga Pathirannehelage and Dr. Jenny J. Yang.</p>

							<h5>Related Resources</h5>

							<p>Click <a href="https://calciomics.org/resources">here</a> to see a list of related resources, including databases and prediction algorithms.</p>

							<h5>Acknowledgments, Publications</h5>

							<p>We would like to acknowledge the efforts of the developers, graduate students and faculty who contributed to the development of these algorithms. The list of related publications is summarized below.</p>

							<h6>Publications</h6>

							<p><i>Yubin Zhou, Shenghui Xue, and Jenny J. Yang. Calciomics: integrative studies of Ca2+ binding proteins and their interactomes in biological systems. Metallomics, (2013) Jan;5(1):29-42. doi: 10.1039/c2mt20009k. PMID: 23235533</i></p>
							<p><i>Kun Zhao, Xue Wang, Michael Kirberger, Hing Wong, Guantao Chen, and Jenny J. Yang, Predicting Calcium-binding Site Using Second Shell Carbon Atoms. Proteins (2012).</i></p>
								
							<p><i>Xue Wang, Michael Kirberger, Guantao Chen, and Jenny J. Yang, Towards Predicting Ca(2+)–binding Sites with Different Coordination Numbers in Proteins with Atomic Resolution. Proteins (2009).</i></p>
							<p><i>Yubin Zhou, Wei Yang, Michael Kirberger, Hsiau-Wei Lee, Gayatri Ayalasomayajula, and Jenny J. Yang. Prediction of EF-hand Calcium Binding Proteins and Analysis of Bacterial EF-hand Proteins. Proteins (2006).</i></p>
								
							<p><i>Hai Deng, Guantao Chen, Wei Yang, Jenny J. Yang. Predicting calcium-binding sites in proteins - a graph theory and geometry approach. Proteins (2006).</i></p>
								
							<p><i>Wei Yang, Hsiau-wei Lee, Homme Hellinga and Jenny J. Yang. Identification and Design of Metal-Binding Proteins. Proteins (2002).</i></p>
								
							<p><i>Wei Yang, Hsiauwei Lee, Homme Hellinga , Michelle Pu and Jenny Jie Yang. Design Calcium Binding Sites by Computer Algorithm. Computational Studies, Nanotechnology , and Solution Thermodynamics of Polymer Systems . Kluwer Academic/Plenum Publishers (2000).</i></p>
								
							<p><i>Mashiku, M., & Edirisinghe, N. (2019, July). Serverless Science Gateway Development for Ca2+ binding site prediction on Amazon Web Services: Case Study. In Proceedings of the Practice and Experience in Advanced Research Computing on Rise of the Machines (learning) (p. 56). ACM.</i></p>

						</div>
					</section>
				</article>

				<!-- Footer
					<footer id="footer">
						<ul class="icons">
							<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
							<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
							<li><a href="#" class="icon brands fa-dribbble"><span class="label">Dribbble</span></a></li>
							<li><a href="#" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
						</ul>
						<ul class="copyright">
							<li>&copy; Untitled</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
						</ul>
					</footer> -->
					<footer id="footer">
						<div class="container">
							<span class="text-light">Developed by <a class="link-light" href="https://arctic.gsu.edu">ARCTIC</a></span>
						</div>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="static/calciomics/assets/js/jquery.min.js"></script>
			<script src="static/calciomics/assets/js/jquery.scrollex.min.js"></script>
			<script src="static/calciomics/assets/js/jquery.scrolly.min.js"></script>
			<script src="static/calciomics/assets/js/browser.min.js"></script>
			<script src="static/calciomics/assets/js/breakpoints.min.js"></script>
			<script src="static/calciomics/assets/js/util.js"></script>
			<script src="static/calciomics/assets/js/main.js"></script>

	</body>
</html>